components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.common/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/common/TenantManagementConstants.java

```diff
@@ -64,7 +73,11 @@ public enum ErrorMessage {
                                                         "Unable to check availability of domain.",
                 "Server encountered an error while checking for tenant domain"),
         ERROR_CODE_DELETE_TENANT_METADATA("TM-65008", "Error while deleting the tenant metadata.",
-                "Server encountered an error while deleting the tenant metadata identified by %s .");
+                "Server encountered an error while deleting the tenant metadata identified by %s ."),
+        ERROR_CODE_ERROR_RETRIEVING_OWNER("TM-65009", "Error while retrieving the owner.",
+                "Server encountered an error while retrieving the owner identified by %s ."),
+        ERROR_CODE_ERROR_UPDATING_OWNER("TM-65010", "Error while updating the owner.",
```

Comment:
identified by the tenant id:

Reply:
Fixed in https://github.com/wso2/identity-api-server/pull/704/commits/cab5ddcccfde9ccfdfaf4a3f123ad205331f331f